frontend/public/module/k8s/cluster-settings.ts

```diff
@@ -240,21 +240,43 @@ export const getReportBugLink = (cv: ClusterVersionKind): { label: string; href:
     return null;
   }
 
-  // Show a Bugzilla link for prerelease versions and a support case link for supported versions.
   const { major, minor, prerelease } = parsed;
-  const bugzillaVersion = major === 4 && minor <= 3 ? `${major}.${minor}.0` : `${major}.${minor}`;
-  const environment = encodeURIComponent(`Version: ${version}
-Cluster ID: ${cv.spec.clusterID}
-Browser: ${window.navigator.userAgent}
-`);
+  let productName;
+  switch (window.SERVER_FLAGS.branding) {
+    case 'openshift':
+    case 'ocp':
+      productName = 'OpenShift Container Platform';
+      break;
+    case 'online':
+      productName = 'OpenShift Online';
+      break;
+    case 'dedicated':
+      productName = 'OpenShift Dedicated';
+      break;
+    case 'azure':
+      productName = 'Azure Red Hat OpenShift';
+      break;
+    default:
+      productName = 'OKD';
+  }
+
+  // Do not show a link for OKD until the new OKD Jira project is ready.
```

Comment:
Waiting for confirmation this is the desired behavior.

Reply:
Confirmed by Michelle Krejci.  See https://coreos.slack.com/archives/C6A3NV5J9/p1658845554775669?thread_ts=1658754855.051769&cid=C6A3NV5J9